prometheus/metrics.go

```diff
@@ -21,13 +21,15 @@ var (
 // MetricsQuery is a common struct for ServiceMetricsQuery and NamespaceMetricsQuery
 type MetricsQuery struct {
 	v1.Range
-	Version      string
```

Comment:
Why is version removed? Don't we still want to be able to get metrics on a specific service?

Reply:
I think it's safer to identify a versioned app/service from its workload name than from the version label, because the latter is optional.